### DISCUSSION TITLE
highscores not submitted?

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#20348] [Q&A]
Sorry if this is a duplicate of some sort, I've noticed on this particular map (might be more I didn't go look for it), I've got some higher scores in local scores that are seemingly not submitted as they didn't end up on the global LB and I couldn't figure out why this is.

Map link: https://osu.ppy.sh/beatmapsets/1222543#osu/3133715
![Screenshot 2022-09-16 at 19 42 18](https://user-images.githubusercontent.com/7279624/190709331-a1452986-5cc9-438c-a968-89ca4399fdeb.jpg)
![Screenshot 2022-09-16 at 19 43 03](https://user-images.githubusercontent.com/7279624/190709451-ae7b6953-a7d8-4d9c-8b1e-83fb2c44fb45.jpg)

**Comment by ghost** [comment]
The scores on the server were calculated before the difficulty adjust mod multiplier was changed, however when it displays the score on your client it recalculates the score

So when you submit a new high score, it compares it to the score you set without the mod multiplier change

**Reply by integerrr** [reply]
no the DA score was 800k before recalc, 400k is correct
